Resources/Private/JavaScript/media-module/src/components/Dialogs/CreateTagDialog.tsx

```diff
@@ -66,7 +66,7 @@ const CreateTagDialog: React.FC = () => {
                     type="text"
                     value={dialogState.label}
                     onChange={setLabel}
-                    onEnterKey={handleCreate}
+                    onEnterKey={createPossible ? handleCreate : null}
```

Comment:
Thx for the improvemt! But did you forget to set the disabled state like you mentioned in the description of the PR?

Reply:
With disable state I meant disabling the button at the beginning. This is done by setting createPossible to something other than true.

Reply:
Ah sorry, you're right. The variable was already there and in use but just not dynamic.